requirements.txt

```diff
@@ -1,2 +1,3 @@
-boto3==1.9.219
-elasticsearch7==7.13.0
\ No newline at end of file
+pytest==5.1.1
+black==22.10.0
+pylint==2.15.5
```

Comment:
These are the requirements for testing the application. There is a separate file for requirements for production.